schemas/2015-10-01/Microsoft.Media.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-10-01/microsoft.media.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Media",
+  "description": "Microsoft Media Service Resource Type",
+  "resourceDefinitions": {
+    "mediaServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Media/mediaServices"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-10-01"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-z0-9]{3,26}$"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/storageAccounts"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "location"
+      ],
+      "description": "Microsoft.Media/mediaServices"
+    }
+  },
+  "definitions" : {
+    "storageAccounts": {
+      "properties": {
+        "storageAccounts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/storageAccount"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "storageAccount": {
+      "properties": {
+        "id": {

```

Comment:
Only the schema should be good. You can push the change to the BrianBlum\master branch and update this PR.